docs/structure-of-a-contract.rst

```diff
@@ -109,7 +109,7 @@ Struct Types
 =============
 
 Structs are custom defined types that can group several variables (see
-:ref:`structs` in types section).
+:ref:`structs` in types section) and can be accessed by single name
```

Comment:
Please use full stops to end sentences.

Do you want to highlight "single" in any way?